### ISSUE TITLE
Separate logic and representation in saved file

### ISSUE THREAD

**Issue by linonetwo** [FlydeHQ/flyde#26]
(as [discussed in discord](https://discord.com/channels/760229165856587817/886465412156493834/1123605291469127733) )

Why both noflo and flyde not use .fbp file anymore?
noflo it use json for saved graph https://github.com/flowhub/the-graph/blob/master/examples/assets/photobooth.json.js
and flyde use yaml https://github.com/FlydeHQ/flyde/blob/main/website/src/pages/playground/_flows/fibonacci.flyde
This mixes the graph functionality with the node position, make it harder for AI to auto generate a graph
I hope we can separate the node position metadata from functionality, so positions saved in a JSON, and logic store in a .fbp file.

But this can be solved by writing a .fbp to yml or json transformer, and use https://github.com/noflo/klayjs-noflo to auto-assign position to nodes

**Comment by GabiGrin** [comment]
`.fbp file:`
I didn't see conforming to an existing format as a leading value in creating Flyde. I started before I discovered the existing work on this subject 🙃   I picked YAML as a starting point but I do see value in converging into a Flyde-specific format in the future. Both to make it even simpler and easier to write by hand, and to alleviate AI-generated content.

Regarding position meta-data, I agree it's not ideal but I aimed for simplicity, and having everything under the same file is the simplest solution.

Once there's a working auto-layout mechanism (for example, with KLayJS), positions could be optional and that could solve this problem.
